tasks/test.py

```diff
@@ -1168,3 +1170,115 @@ def get_modified_files(ctx):
 
     modified_files = ctx.run(f"git diff --name-only {last_main_commit}", hide=True).stdout.splitlines()
     return modified_files
+
+
+@task
+def send_unit_tests_stats(_, job_name):
+    fast_success = True
+    classic_success = True
+
+    series = []
+
+    failed_tests_classic, n_test_classic = parse_test_log("test_output.json")
```

Comment:
there is a risk of breaking everything if test name changes, but it's maybe ok

Reply:
Yes I think the test name should not change during the time where this invoke task is used